system_strings/sysmes_text.en

```diff
@@ -1189,23 +1189,23 @@ I leave the sleeping Arcueid and^exit the apartment alone, heading^for the depar
 アルクェイドもほっとけないが、^シエル先輩も大切なひとだ。^敵とは思いたくはない。
 新天地を求め集まった若者達。^振り降ろされた凶器は容赦なく、^彼らの体を圧し潰してゆく―――
 まるで寝付く事ができない。^今夜も本を手に取った。^読んでいれば眠気も来るだろう。
-何もかもがあり得ない事ばかりで^頭が上手く働いてくれない。^思い返すとしたら、それは……。
-アルクェイドを思い返した。^昨夜の表情が気にかかる。^傷が完治してないのだろうか？
-影の男を思い返していた。^自分を狙って来たのだろうか？^だが、それも終わった事だ。
-先生……ノエルを思い返す。^シスター……？^まあシスターかな……。
-まずは朝食を摂らなければ、と^ベッドから起き上がる。^だが、体の不調は―――
-学校を休む事にした。^翡翠を安心させるためにも、^診察を受けておこうか……？
-阿良句博士にお願いしよう。^いまいち安心できない人だが、^あれでも専門家には違いない。
-琥珀さんにお願いしよう。^薬を処方してもらえれば、^あとは自力で回復する。
-部屋にいてもやるべき事がない。^体の調子は戻ってきたのだから、^ここはひとつ―――
-子供の時以来となる中庭の建物。^そこには七年前に彫られたらしい^奇妙な落書きが……。
-西棟二階には槙久の書斎がある。^ふと、昨夜のアルクェイドから^言われた事を思い出していた。
-今、ここには自分しかいない。^亡き父親の部屋を調べるには、^またとない好機だった。
-書斎に入るのは止めておく。^答えが見つかる確証もない。^気分を変えて一階に向かおう。
-――――――。
-言い争う声で目が覚めた。^ベッドに寝かされた自分の体が、^思うように動かせない―――
-このまま無理をすると、^秋葉を泣かせてしまう気がした。^今日の所は大人しく寝ていよう。
-約束を破る事はできない。^どうにか秋葉達の目を盗んで、^公園まで行かなければ―――
+Everything is so ridiculous I'm^unable to process it properly.^Thinking back on it...
+I think back on Arcueid. I'm^worried about her expression^from last night. Have her wounds^not fully healed yet?
+I think back on the shadowy^figure. Was he targetting me?^Well, not like I'll be seeing^it again.
+Sensei... I think back on Noel.^Is she a sister? Well, I guess^she's a sister.
```

Comment:
Ellipsis should be moved here imo
> I think back on Noel...Sensei? 

Also I'd be inclined to translated sister as nun. Seems like a false friend loanword to me. Check what we did in the script though.

Reply:
We used both
https://github.com/Tsukihimates/Tsukihime-Translation/blob/a8db6e13504762bfe5707d3b3dda8ab0f47747c1/script/Arcueid/Day%202/02_00_ARC02_1_1.txt#L209
https://github.com/Tsukihimates/Tsukihime-Translation/blob/a8db6e13504762bfe5707d3b3dda8ab0f47747c1/script/Arcueid/Day%208/08_00_ARC08_8.txt#L834